src/partitioning/loadBalance.F90

```diff
@@ -591,32 +593,27 @@ subroutine blockDistribution
     if(commNeglected .and. myID == 0) then
        print "(a)", "#"
        print "(a)", "#                    Warning"
-       print "(a)", "# Communication costs neglected to obtain&
-            & a valid partitioning."
+       print "(a)", "# Communication costs neglected to obtain a valid partitioning."
     endif
 
     ! If the load imbalance tolerance was not met, print a warning
     ! message if I am processor 0.
 
+    loadFormat = "(*(A, F6.3))"
+
     if (myid == 0) then
        if(.not.(cellsBalanced .and. facesBalanced)) then
           print "(a)", "#"
           print "(a)", "#                    Warning"
-          print 100, loadImbalance
-          print 101, ubvec(1), ubvec(2)
+          write(*, loadFormat) "# Specified load imbalance tolerance", real(loadImbalance), " not achieved."
```

Comment:
I used write because Modern Fortran in VSCode had weird syntax highlighting when I used print with a variable format. The code compiles and runs fine though.
![print_highlight](https://user-images.githubusercontent.com/48863473/213276227-85344285-7e2e-49fd-904f-c7ac1a62c946.png)